packages/ckeditor5-ui/docs/_snippets/features/blocktoolbar.js

```diff
@@ -125,6 +125,9 @@ BalloonEditor
 		toolbar: {
 			items: [ 'bold', 'italic', '|', 'link' ]
 		},
+		ckbox: {
+			forceDemoLabel: true
+		},
```

Comment:
In this demo I don't see the `insert with file manager` toolbar button.

Reply:
In commit.